src/components/Signin.vue

```diff
@@ -1,134 +1,99 @@
+  </div>
+  </div>
+
+</template>
+
+<script>
+import axios from 'axios';
+
+export default {
+data() {
+return {
+  name: '',
+  email: '',
+  password: '',
+  submitButtonText: 'Sign In ',
+  errorMessage:"",
+  formErrors: {},
+};
+},
+
+methods: {
+
+submitForm(e) {
+ e.preventDefault();
+  axios.post("http://localhost:8080/api/users/signin", {
+          email: this.email,
+          password: this.password
+  },
+  {
+			headers: {
+				"Content-Type":"application/json"	
       }
-    }
-  }
-  </script>
+			})
+    .then(response => {
   
-  <style lang="scss" scoped>
-  .container {
-    position: relative;
-    width: 500px;
-    height: 480px;
-    border-radius: 10px;
-    overflow: hidden;
-    box-shadow: 0 15px 30px rgba(0, 0, 0, .2),
-      0 10px 10px rgba(0, 0, 0, .2);
-    background: linear-gradient(to bottom, #efefef, #ccc);
-    .overlay-container {
-      position: absolute;
-      top: 0;
-      left: 50%;
-      width: 50%;
-      height: 100%;
-      overflow: hidden;
-      transition: transform .5s ease-in-out;
-      z-index: 100;
-    }
-    @mixin overlays($property) {
-      position: absolute;
-      top: 0;
-      display: flex;
-      align-items: center;
-      justify-content: space-around;
-      flex-direction: column;
-      padding: 70px 40px;
-      width: calc(50% - 80px);
-      height: calc(100% - 140px);
-      text-align: center;
-      transform: translateX($property);
-      transition: transform .5s ease-in-out;
-    }
-  }
-  h2 {
-    margin: 0;
-  }
-  p {
-    margin: 20px 0 30px;
-  }
-  a {
-    color: #222;
-    text-decoration: none;
-    margin: 15px 0;
-    font-size: 1rem;
-  }
-  button {
-    border-radius: 20px;
-    border: 1px solid #009345;
-    background-color: #009345;
-    color: #fff;
-    font-size: 1rem;
-    font-weight: bold;
-    padding: 10px 40px;
-    letter-spacing: 1px;
-    text-transform: uppercase;
-    cursor: pointer;
-    transition: transform .1s ease-in;
-    &:active {
-      transform: scale(.9);
-    }
-    &:focus {
-      outline: none;
-    }
-  }
-  form {
-    position: absolute;
-    top: 0;
-    display: flex;
-    align-items: center;
-    justify-content: space-around;
-    flex-direction: column;
-    padding: 100px 1px;
-    width: calc(33%);
-    height: calc(100%);
-    text-align: center;
-    background: linear-gradient(to bottom, #efefef, #ccc);
-    transition: all .5s ease-in-out;
-    div {
-      font-size: 1rem;
-    }
-    input {
-      background-color: #eee;
-      border: none;
-      padding: 5px 15px;
-      margin: 1px 0;
-      width: calc(100%);
-      border-radius: 15px;
-      border-bottom: 1px solid #ddd;
-      box-shadow: inset 0 1px 2px rgba(0, 0, 0, .4),
-        0 -1px 1px #fff,
-        0 1px 0 #fff;
-      overflow: hidden;
-      &:focus {
-        outline: none;
-        background-color: #fff;
+      if (response.status==200){
+        localStorage.setItem("user-info",JSON.stringify(response.data))
```

Comment:
Only need to set userId

Reply:
userid is user-info.id

Reply:
You need to use json.parse when access the data. You also leave the password for other to see when setting the whole object. You should set userId and username as key value pair only